docs/pages/Applications/Particle_Mechanics_Application/Examples/menu_info.json

```diff
@@ -4,7 +4,7 @@
       "type": "kratos_example",
       "raw_url": "https://raw.githubusercontent.com/KratosMultiphysics/Examples/master/particle_mechanics/validation/granular_flow_2D/README.md",
       "source_url": "https://github.com/KratosMultiphysics/Examples/blob/master/particle_mechanics/validation/granular_flow_2D/README.md",
-      "file_name": "Granular_flow_2d_readme.md"
```

Comment:
I'm curious the reasoning behind the change in file names. Is there a specific one?